internal/utils/protocov/protocov.go

```diff
@@ -12,30 +12,30 @@ import (
 	"google.golang.org/protobuf/types/known/anypb"
 )
 
-const (
-	APIPrefix = "type.googleapis.com/"
-)
-
-var marshalOpts = proto.MarshalOptions{}
+// Deprecated: error should not be ignored, use ToAnyWithValidation instead.
+func ToAny(msg proto.Message) *anypb.Any {
+	res, err := ToAnyWithValidation(msg)
+	if err != nil {
+		return nil
```

Comment:
The validation error should not be ignored. 

Deprecated the `ToAny` function and will clean up all the refrences to it in a follow-up PR.

Reply:
for case like following, ToAny still meanfull.
```console
GRPCWeb = &hcm.HttpFilter{
		Name: wellknown.GRPCWeb,
		ConfigType: &hcm.HttpFilter_TypedConfig{
			TypedConfig: protocov.ToAny(&grpcweb.GrpcWeb{}),
		},
	}
```

Reply:
should we change the comment?

Reply:
I prefer removing this function at all, but let's discuss and do it in a follow-up PR.